common/definitions.py

```diff
@@ -2276,3 +2276,5 @@
                            'subunit gamma':'G&gamma;'}
 
 ARRESTIN_DISPLAY_NAME = {'arrs':'S-arrestin', 'arrb1':'Beta-arrestin-1'}
+
+CLASSLESS_PARENT_GPCR_SLUGS = set({'008'}) # name = Other GPCRs
```

Comment:
this is going to be slug '010' in the new build. Some other class slugs have also changed

Reply:
I added a fix for CLASSLESS_PARENT_GPCR_SLUGS definition changes at [3a9af06](https://github.com/protwis/protwis/pull/550/commits/3a9af0650ba6e583822e7c70772189c70454f22a).

Reply:
I tested the fix against the new build DB after running build_ligand_search.py. The CLASSLESS_PARENT_GPCR_SLUGS gets correctly and automatically set up to
CLASSLESS_PARENT_GPCR_SLUGS = {'010'}.

I have to update yet the build for GPCR similarities (not related with this PR).